src/devices/wifi/nm-device-iwd.c

```diff
@@ -1109,7 +1109,7 @@ wifi_secrets_cb (NMActRequest *req,
 	cleanup_association_attempt (self, TRUE);
 }
 
-static bool
+static void
```

Comment:
you did the fixups on the second commit, but they really belong to the first one. Could you rebase the branch so that the second commit has the quit simple change for -1 to G_MAXINT?

While at it, could you rebase the branch on master?

The rest looks good to me.

Reply:
Oops, yes, fixing.